internal/runtime/metrics/metrics.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package metrics provides functions for creating runtimes SDK related metrics.
```

Comment:
```suggestion
// Package metrics provides functions for creating Runtime SDK related metrics.
```

And elsewhere - good to be consistent with capitalization etc.